docs/ci-services.md

```diff
@@ -1,45 +1,50 @@
 # CI Services
 
-The CI service blob defines the supported CI services the user will be
+The CI services map defines the supported CI services the user will be
 presented with when running the scaffolder.
 
 ## Example
 
 ```javascript
 {
-  TravisCI: {scaffolder: travisScaffolder, public: true}
+  CIServiceName: {scaffolder: opts => scaffold(opts), public: true}
 }
 ```
 
 ## Config
 
 ### keys: __string__
 
-Keys in the CI services blob define the options the user will be presented with
-when choosing their desired CI service.
+Keys in the CI services map define the choices the user will be presented with
+when choosing their desired CI service. These have no functional impact on the
+scaffolder.
 
-example: `TravisCI`
+example: `CIServiceName`
 
 ### values: __object__
 
-Values in the CI services blob define the actual mechanisms and configuration
+Values in the CI services map define the actual mechanisms and configuration
 that will be used by the JavaScript Scaffolder to scaffold the CI service for
 the user's project.
 
 #### scaffolder: __function__ (_required_)
 
-Consumer-defined plugin(??) called when this ci service is selected by the
-user in the CLI. Will be called with the following arguments:
+Consumer-provided plugin(??) called when this CI service is selected by the
+user in the interactive prompt. Will be called with the following arguments:
 
-* `projectRoot` __string__ full path to directory where scaffolder was initiated
+* `projectRoot` __string__ full path to directory where scaffolder will create
+  the JavaScript project
 * `vcs` __object__
   * `host` __string__ VCS hosting service to generate the repo with (e.g., `github`)
   * `owner` __string__ account name on the host service for the repository to
     generate (e.g., `form8ion`)
   * `name` __string__ repository name to generate (e.g., `javascript-scaffolder`)
-* `visibility` __string__ visibility of the project to be scaffolded (e.g., `Public`, `Private`)
-* `projectType` __string__ type of project to be scaffolded (e.g., `Application`, `Package`, `CLI`)
-* `nodeVersionCategory` __string__ node category chosen for the project (e.g., `LTS`)
+* `visibility` __string__ visibility of the project to be scaffolded (valid
+  values are: `Public`, `Private`)
```

Comment:
👍🏼 i like this. 

i also look at it and think "how are we going to remember to update this when we add another valid value". might be the type of thing that could be small enough to generate at some point as a way to dip out toe into the process

Reply:
agreed. i'm not sure how we'll keep this all up to date confidently. linking to more "general" core shapes that we pull out elsewhere like you mentioned in another comment will help a lot in that realm i think.